### DISCUSSION TITLE
Setting Colors for the Ref Sequence

### DISCUSSION THREAD

**Discussion by Vishvak2000** [GMOD/jbrowse-components#2082] [Q&A] [answered]
Hello! I am trying to configure a browser with a viral protein sequence as the reference sequence. Is there any way to configure the browser to recognize the reference sequence (which is a FASTA amino acid sequence) as a protein sequence i.e., is there any way to NOT color the A,T,C,Gs in the reference sequence?

Thanks!

**Comment by garrettjstevens** [comment, marked as answer]
JBrowse assumes the reference sequence is DNA, not protein. As a hack, you could try adjusting the theme so that A, T, C, and G have the same color as an unknown base. That would involve putting the theme in the configuration in the top level of your config file:

```json
{
  "configuration":{
    "theme": {
      "palette": {
        "bases": {
          "A": {
            "main": "#aaa"
          },
          "C": {
            "main": "#aaa"
          },
          "G": {
            "main": "#aaa"
          },
          "T": {
            "main": "#aaa"
          }
        }
      }
    }
  }
}
```

There's no guarantee that other things would work with a protein reference sequence, though.

**Reply by cmdcolin** [reply]
would it be allowed for them to specify non-ACGT bases here?

**Reply by garrettjstevens** [reply]
Yeah, good idea, that does work. I didn't even think about it. You'd also want to turn off "Show reverse" and "Show translation" for the track, too, since those don't make sense with a protein reference.
